src/Sentry.Protocol/BaseScopeExtensions.cs

```diff
@@ -170,7 +165,7 @@ public static void SetFingerprint(this BaseScope scope, IEnumerable<string> fing
         /// <param name="key">The key.</param>
         /// <param name="value">The value.</param>
         public static void SetExtra(this BaseScope scope, string key, object value)
-            => ((ConcurrentDictionary<string, object>)scope.Extra).AddOrUpdate(key, value, (s, o) => value);
+            => ((ConcurrentDictionary<string, object>) scope.Extra).AddOrUpdate(key, value, (s, o) => value);
```

Comment:
Are these spaces and `.` on comments added automatically, on save?
These improvements are great but makes this PR a bit harder to read (1k+/1k-)

Maybe for the next PRs they can be split? If it's an automated thingy, feel free to run it on the whole codebase, we can merge without reviewing (more than skimming)

Reply:
Sorry, this one is automatic by Rider. I'll look through the changes and revert formatting changes to avoid noise.